src/utils/utils.ts

```diff
@@ -50,7 +50,7 @@ type OutputObject =
       data: Error;
     };
 
-function standardErrorReplacer(key: string, value: any) {
```

Comment:
How come the `_key` is not used here?

Reply:
resolved in 84e7c348f84138787b90ead0a47528b9af789b7d